custom_components/localtuya/__init__.py

```diff
@@ -49,30 +45,6 @@
 }
 
 
-def prepare_setup_entities(config_entry, platform):
-    """Prepare ro setup entities for a platform."""
-    entities_to_setup = [
-        entity
-        for entity in config_entry.data[CONF_ENTITIES]
-        if entity[CONF_PLATFORM] == platform
-    ]
-    if not entities_to_setup:
-        return None, None
-
-    tuyainterface = pytuya.TuyaInterface(
-        config_entry.data[CONF_DEVICE_ID],
-        config_entry.data[CONF_HOST],
-        config_entry.data[CONF_LOCAL_KEY],
-        float(config_entry.data[CONF_PROTOCOL_VERSION]),
-    )
-
-    for interface_config in entities_to_setup:
-        # this has to be done in case the device type is type_0d
-        tuyainterface.add_dps_to_request(interface_config[CONF_ID])
-
-    return tuyainterface, entities_to_setup
```

Comment:
For the sake of clarity, would name it entity_config, rather than interface_config.

Reply:
Agree!